<?xml version="1.0" ?>
<launch>
  <arg name="robot_ip" />
  <arg name="arm_id"      default="panda" />
  <arg name="joint_names" default="[$(arg arm_id)_finger_joint1, $(arg arm_id)_finger_joint2]" />
  <arg name="config_file" default="$(find lr_panda)/config/franka_gripper_config.yaml"/>

  <node name="franka_gripper" pkg="franka_gripper" type="franka_gripper_node" output="screen">
    <param name="robot_ip" value="$(arg robot_ip)"/>
    <rosparam command="load" file="$(arg config_file)" />
    <rosparam param="joint_names" subst_value="true">$(arg joint_names)</rosparam>
  </node>

</launch>